renormalizer/mps/mps.py

```diff
@@ -199,38 +207,24 @@ def hartree_product_state(cls, mol_list, condition):
         
         qn_single = [[],] * mol_list.nsite
         
-        for v_dof in mol_list.v_dofs:
+        for v_dof in mol_list.e_dofs + mol_list.v_dofs:
             isite = mol_list.order[v_dof]
             pdim = mol_list.basis[isite].nbas
             ms = np.zeros((1, pdim, 1))
             local_state = condition.get(v_dof,0)
-            ms[0, local_state, 0] = 1.
-            mps[isite] = ms
-            qn_single[isite] = mol_list.basis[isite].sigmaqn[local_state]
-        
-        if not mol_list.multi_dof_basis:
-            for e_dof in mol_list.e_dofs:
-                isite = mol_list.order[e_dof]
-                pdim = mol_list.basis[isite].nbas
-                ms = np.zeros((1, pdim, 1))
-                local_state = condition.get(e_dof,0)
+            if isinstance(local_state, int):
                 ms[0, local_state, 0] = 1.
```

Comment:
It seems that the Hartree product state of  `BasisMultiElectronVac` should be taken with great care because a `gs` is added. Maybe we should add some notes in the doc.